src/views/index.mustache

```diff
@@ -791,3 +845,9 @@
     </div>
 
 {{> footer}}
+
+<script type="text/javascript">
+$(function () {
+    $('[data-toggle="popover"]').popover();
+})
+</script>
```

Comment:
It's best to move this to `index.js`.

Reply:
I need to remove this completely actually, it's not being used, couldn't get the popover to show with being stacked on multiple modals.